dnf5/commands/download/download.cpp

```diff
@@ -115,6 +118,12 @@ void DownloadCommand::set_argument_parser() {
             return true;
         });
 
+    auto urlallmirrors = parser.add_new_named_arg("all-mirrors");
```

Comment:
I would use "allmirros" witout a hypen. `--urlprotocol` is also spelled without a hyphen.